packages/javascript/src/span.ts

```diff
@@ -43,12 +52,12 @@ export class Span extends Serializable<AppsignalSpan> {
   }
 
   public setTags(tags: object): this {
-    this._data.tags = { ...this._data.tags, ...tags }
+    this._data.tags = { ...this._data.tags, ...sanitizeParams(tags) }
     return this
   }
 
   public setParams(params: object): this {
-    this._data.params = { ...this._data.params, ...params }
+    this._data.params = { ...this._data.params, ...sanitizeParams(params) }
```

Comment:
It shouldn't be necessary for params, right? We support non-string types here.

Reply:
Not in this first iteration, this is the struct that the JSON gets parsed into server-side. 

```
#[derive(Debug,Clone,Deserialize,PartialEq)]
pub struct FrontendTransaction {
    timestamp: i64,
    action: Option<String>,
    namespace: String,
    error: FrontendError,
    revision: Option<String>,
    #[serde(rename = "sourcemapUrl")]
    sourcemap_url: Option<String>,
    tags: Option<HashMap<String,String>>,
    params: Option<HashMap<String,String>>,
    environment: Option<HashMap<String,String>>,
}
```

Reply:
Ok so we will support it in the future?

Reply:
Most likely, I'll take a look next week to see if we can do this easily.

Reply:
ok!